build.sbt

```
@@ -111,6 +108,12 @@ lazy val zhttpBenchmarks = (project in file("./zio-http-benchmarks"))
       ),
   )
 
+// Testing Package
+lazy val zhttpTest = (project in file("./zio-http-test"))
+  .dependsOn(zhttp)
+  .settings(stdSettings("zhttp-test"))
```

```suggestion
  .settings(stdSettings("zhttpTest"))
```

To be consistent with `zhttpBenchmarks`